src/components/NavBar.vue

```diff
@@ -1,31 +1,113 @@
 <template>
-	<nav>
+	<div id="nav-bar">
```

Comment:
Could we use the `<header>` tag here? A bit more semantically correct for SEO

Reply:
I like it, will do